modules/tensor_mechanics/src/actions/TensorMechanicsAxisymmetricRZAction.C

```diff
@@ -63,20 +61,15 @@ TensorMechanicsAxisymmetricRZAction::act()
   if (isParamValid("base_name"))
     params.set<std::string>("base_name") = getParam<std::string>("base_name");
 
-  std::string short_name = "TensorMechanicsRZ";
-
-
   for (unsigned int i = 0; i < dim; ++i)
   {
-    std::stringstream name;
-    name << short_name;
-    name << i;
+     // Create kernal name dependent on the displacement variable
+     std::string kernel_name = "TensorMechanicsAxisymmetricRZ" + _base_name + Moose::stringify(i);
 
-    params.set<unsigned int>("component") = i;
-    params.set<NonlinearVariableName>("variable") = displacements[i];
-    params.set<std::vector<AuxVariableName> >("save_in") = save_in[i];
+     params.set<unsigned int>("component") = i;
+     params.set<NonlinearVariableName>("variable") = displacements[i];
+     params.set<std::vector<AuxVariableName> >("save_in") = save_in[i];
 
-    _problem->addKernel("StressDivergenceRZTensors", name.str(), params);
+     _problem->addKernel("StressDivergenceRZTensors", kernel_name, params);
```

Comment:
You're using Atom right? Your editor should format for you. I know in Emacs
we can highlight a whole region of code and tell it to indent it correctly.
Atom probably does this too.

On Tue, Mar 1, 2016 at 12:15 PM Stephanie Pitts <notifications@github.com>
wrote:

> In
> modules/tensor_mechanics/src/actions/TensorMechanicsAxisymmetricRZAction.C
> <https://github.com/idaholab/moose/pull/6380#discussion_r54618829>:
>
> >
> > -    _problem->addKernel("StressDivergenceRZTensors", name.str(), params);
> > +     _problem->addKernel("StressDivergenceRZTensors", kernel_name, params);
>
> Oh man, sad days. I've never spent so much time counting spaces before
> trying to submit Moose code.
>
> I'll wait a bit for other comments to accumulate before pushing to correct
> the whitespace.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/6380/files#r54618829>.
>